sdk/core/azure-common/CHANGELOG.md

```diff
@@ -65,6 +65,8 @@
 
 Thanks to patrikn for the contribution
 
+[comment]: # ( cspell:ignore patrikn )
```

Comment:
hey @wonhyeongseo thanks for your PR! Can we move all the inline cspell ignores in the PR to the cspell.json under filtered paths? We try to keep the cspell statements out of our src code / docs. Thanks!